Modix.Data/Models/EphemeralUser.cs

```diff
@@ -0,0 +1,111 @@
+﻿using Discord;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Modix.Data.Models
+{
```

Comment:
Would this fit better in Modix.Services? I don't like the idea of having Modix.Data have dependencies to Discord. And really, it only exists to simplify the service layer, so IGuildUser can be passed around in more scenarios, but the service layer should translate it all over to GuildUserEntity etc. to interact with Modix.Data.

Reply:
Hmm, a good point. I'll move it.